src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -2,11 +2,15 @@
 
 {% load staticfiles %}
 
+{% block metatitle %}
+    <title>Model My Watershed Micro Site Storm Model</title>
```

Comment:
I think this might look better with some sort of separator between the main title and the sub, `:`, `-`, `|` or something.  Hope those don't just look like emojis here.